docs/development/add-data-guide.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[add-data-guide]]
+== Add Data guide
+
+`Add Data` in the Kibana Home application contains a collection of tutorials for setting up data flows in the Elastic stack.
+
+Each tutorial contains 3 sets of instructions
+
+1. `On Premise` Instructions for setting up a data flow when both Kibana and Elasticsearch are running on premise
+2. `On Premise Elastic Cloud` Instructions for setting up a data flow when Kibana is running on premise but
+Elasticsearch is running on cloud.
+3. `Elastic Cloud` Instructions for setting up a data flow when both Kibana and Elasticsearch are running on cloud.
+
+[float]
+=== Creating a new tutorial
+1. Create a new directory in the link:https://github.com/elastic/kibana/tree/master/src/core_plugins/kibana/server/tutorials[tutorials directory].
+2. In the new directory, create a file called `index.js` that exports a function.
+The function must return a Javascript object that conforms to the link:https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/common/tutorials/tutorial_schema.js[tutorial schema].
+3. Register the tutorial in link:https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/server/tutorials/register.js[register.js] by calling `server.registerTutorial(myFuncImportedFromIndexJs)`.
+4. Add image assets to the link:https://github.com/elastic/kibana/tree/master/src/core_plugins/kibana/public/home/tutorial_resources[tutorial_resources directory].
+5. Run Kibana locally to preview the tutorial.
+6. Create a PR and go through the review process to get the changes approved.
+
+[float]
+==== Variables
+String values can contain variables that get substituted when rendered. Variables are specified by `{}`.
+For example: `{config.docs.version}` would get rendered as `6.2` when running the tutorial in Kibana 6.2.
```

Comment:
Change : to , and "would get" to "is"

For example`,``{config.docs.version}` `is` rendered as ...